circuitbreakerrpc/circuitbreaker.proto

```diff
@@ -108,4 +114,38 @@ message Counter {
     int64 fail = 1;
     int64 success = 2;
     int64 reject = 3;
-}
\ No newline at end of file
+}
+
+message ListForwardingHistoryRequest {
+    // The inclusive start time for the query, used to filter HTLCs by the time they were added 
+    // to local incoming channel. If this value is zero, it will be treated as the unix epoch.
+    int64 start_time_ns = 1;
```

Comment:
Perhaps add a reference to `add` in the name? `add_time_start` or something similar?